cmd/mt-gateway/main.go

```diff
@@ -47,14 +51,27 @@ func main() {
 		fmt.Println("Flags:")
 		flag.PrintDefaults()
 	}
-	flag.Parse()
+
+	// Only try and parse the conf file if it exists
+	path := ""
+	if _, err := os.Stat(*confFile); err == nil {
+		path = *confFile
+	}
+	conf, err := globalconf.NewWithOptions(&globalconf.Options{
+		Filename:  path,
+		EnvPrefix: "MT_",
```

Comment:
would it make sense to use a different string than `MT_`? IIRC in tsdb-gw we're using `GW_`, maybe that would make more sense since this is also a gateway.

Reply:
I was going back and forth of that...

I think it's pretty likely that we'll be sharing configuration (we're using the same config dir and likely schema files), so we'll probably want to share env vars as well.

I could be convinced though

Reply:
fair enough. at least that would then be consistent with what the documentation says https://github.com/grafana/metrictank/blob/master/docs/config.md#config